news/_posts/2020-02-04-the-new-prestashop-test-framework.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "The (new) PrestaShop Test Framework"
+subtitle: "And how we’re continually improving the quality of the PrestaShop project"
+date:  2020-02-05 12:00:00
+authors: [SimonGarny]
+icon: icon-checked
+image: /assets/images/theme/meta-logo-build.png
+tags: [QA, tests, community]
+---
+
+For a few months, quality has been a priority for PrestaShop: the QA team and I are proud to announce that we’re making great steps in improving the overall quality of the core project. Automatizing tests is a big part of this process: let’s see what we’re doing, why, and how!
+Note: this article is focused on the Core only, but that’s just a small part of what the QA team is working on. More to come in a future article!
+
+
+## Taking a look back
+
+Historically, the PrestaShop Core continuous integration had a big end-to-end test campaign which purpose was to test the whole application as thoroughly as possible. This campaign was using Selenium as the automation tool, and we were running it in two places:
```

Comment:
Would be nice to be able to provide external links to statements about this previous campaign (another Build post ?)